source/user_guide.rst

```diff
@@ -312,22 +281,18 @@ Let's send this message to dojot:
 
   mosquitto_pub -t /admin/0998/attrs -m '{"temperature": 10.6}'
 
-If there is no output, the message was sent to MQTT broker. The topic is build 
-from the following information:
+If there is no output, the message was sent to MQTT broker. The topic is build from the following information:
 
-- admin: user tenant. This is retrieved from "service" attribute from user
-  configuration.
-- 0998: device ID. This is retrieved from the device itself. It is returned
-  when the device is created or read from /device endpoint.
+- admin: user tenant. This is retrieved from "service" attribute from user configuration.
+- 0998: device ID. This is retrieved from the device itself. It is returned when the device is created or read from 
+  /device endpoint.
 
 To check if it was correctly processed by dojot, send the following request:
 
 .. code-block:: bash
     
     curl -X POST http://localhost:8000/metric/v2/entities/0998 \
```

Comment:
This has to be updated, no?

Reply:
I'll add a note indicating that this is the default behavior and it can be changed in device configuration.